client/src/main/java/com/tigrisdata/db/client/search/SearchRequest.java

```diff
@@ -214,17 +215,29 @@ public Builder withFacetQuery(FacetQuery facetQuery) {
     }
 
     /**
-     * Optional - Sets the SortOrder to sorts search results according to specified attributes and
+     * Optional - Sets the SortingOrder to sort search results according to specified attributes and
      * indicated order
      *
-     * @param sortOrder {@link SortOrder}
+     * @param sortOrder {@link SortingOrder}
      * @return {@link SearchRequest.Builder}
      */
-    public Builder withSortOrder(SortOrder sortOrder) {
+    public Builder withSort(SortingOrder sortOrder) {
```

Comment:
What are the use cases when this method is useful vs the following?

Reply:
can you add a rpc method test to exercise the full trip?

Reply:
Allow sharing of `SortingOrder` between search requests. Or if we introduce sorting for read.

Reply:
makes sense.